src/main/java/org/jboss/threads/Version.java

```diff
@@ -48,7 +50,8 @@ private Version() {
         }
         JAR_NAME = jarName;
         VERSION_STRING = versionString;
-        try {
+        boolean logVersion = AccessController.doPrivileged((PrivilegedAction<Boolean>) () -> Boolean.valueOf(Boolean.parseBoolean(System.getProperty("jboss.log-version", "true")))).booleanValue();
```

Comment:
FWIW, in my version (which is missing the privileged thing for sure), I put this part in a try/catch to make sure we keep the current behavior even if something wrong gets in the way:
https://github.com/jbossas/jboss-threads/pull/119/files#diff-f7179ffcce5656c32a94e972f59059723d7d9d1c53bc3274c1d7bdda0b6a212aR57